Mathlib/Data/ENat/Lattice.lean

```diff
@@ -190,11 +190,11 @@ lemma iSup_add_iSup_of_monotone {ι : Type*} [Preorder ι] [IsDirected ι (· 
     (hf : Monotone f) (hg : Monotone g) : iSup f + iSup g = ⨆ a, f a + g a :=
   iSup_add_iSup fun i j ↦ (exists_ge_ge i j).imp fun _k ⟨hi, hj⟩ ↦ by gcongr <;> apply_rules
 
-proof_wanted smul_iSup {R} [SMul R ℕ∞] [IsScalarTower R ℕ∞ ℕ∞] (f : ι → ℕ∞) (c : R) :
-    c • ⨆ i, f i = ⨆ i, c • f i
+-- proof_wanted smul_iSup {R} [SMul R ℕ∞] [IsScalarTower R ℕ∞ ℕ∞] (f : ι → ℕ∞) (c : R) :
+--     c • ⨆ i, f i = ⨆ i, c • f i
 
-proof_wanted smul_sSup {R} [SMul R ℕ∞] [IsScalarTower R ℕ∞ ℕ∞] (s : Set ℕ∞) (c : R) :
-    c • sSup s = ⨆ a ∈ s, c • a
+-- proof_wanted smul_sSup {R} [SMul R ℕ∞] [IsScalarTower R ℕ∞ ℕ∞] (s : Set ℕ∞) (c : R) :
+--     c • sSup s = ⨆ a ∈ s, c • a
 
```

Comment:
It seems `lake shake` does not look at `proof_wanted` and does not want me to add imports which makes `IsScalarTower` available in this file

Reply:
Please add an entry to `scripts/noshake.json` to keep the import anyways.

Also, please report this problem as a github issue: we should fix shake.

Reply:
I need to double check my claim that `shake` does not look at `proof_wanted`, I cannot seem to reproduce the behaviour